usecase/usecase-cordova-android-tests/extra_plugins/cordova-plugin-crosswalk-webview-tests/README.md

```diff
@@ -0,0 +1,26 @@
+## Introduction
+
+This plugin is for checking the crosswalk webview api
+
```

Comment:
This plugin is used for testing the Embedding API in cordova-plugin-crosswalk-webview.